docs/epiviz/studycountry.rst

```diff
@@ -0,0 +1,66 @@
+.. _study-country-covariates:
+
+Study and Country Covariates
+----------------------------
+
+There are three reasons to use a covariate.
```

Comment:
Is it worth mentioning the one covariate? I guess the modelers will never interact with it directly?